roles/php/tasks/main.yml

```diff
@@ -1,23 +1,24 @@
-# Install the "REMI" repo which contain newer php packages that override the default
-# that come with the distro (CentOS7: 5.4; CentOS6: 5.3)
-- name: check for presence of the remi repository file
-  stat:
-    path: "/etc/yum.repos.d/remi.repo"
```

Comment:
Is it now certain that this file is gone?

Reply:
yes, it's in the remi-release package, which is being removed. Yum saves a copy in /etc/yum.repos.d/remi.repo.rpmsave